actionlist/__tests__/reducers/actionItemsSpec.js

```diff
@@ -212,3 +212,40 @@ describe('action_items', () => {
     expect(reducer(beforeState, action)).toEqual(afterState);
   });
 });
+
+describe('action_items', () => {
+  it('should have the colors reset to white on pressing reset button', () => {
+    const beforeState = {
+      dropdownColors: {
+        Saved: preferenceColors.aqua,
+        Initiated: preferenceColors.green,
+        Disapproved: preferenceColors.orange,
+        Enroute: preferenceColors.pink,
+        Approved: preferenceColors.purple,
+        Final: preferenceColors.aqua,
+        Processed: preferenceColors.orange,
+        Exception: preferenceColors.grey,
+        Cancel: preferenceColors.tan,
+      },
+    };
+    const action = resetPreferences();
+
+    const afterState = {
+      dropdownColors: {
```

Comment:
You already export `actionItems.defaultState`, so you might get a better test by checking whether the output of the reducer function matches the default state.